pkgs/leak_tracker_flutter_testing/lib/src/model.dart

```diff
@@ -57,7 +57,36 @@ class LeakTrackingTestConfig {
     this.allowAllNotGCed = false,
   });
 
-  /// Creates a new instance of [LeakTrackingTestConfig] to collect retaining path.
+  /// Creates a new instance for debugging notGCed leaks.
+  ///
+  /// This configuration will collect stack traces on disposal,
+  /// and retaining path for notGCed objects.
+  LeakTrackingTestConfig.debugNotGCed({
+    this.leakDiagnosticConfig = const LeakDiagnosticConfig(
+      collectStackTraceOnDisposal: true,
+      collectRetainingPathForNotGCed: true,
+    ),
+    this.notGCedAllowList = const <String, int>{},
+    this.notDisposedAllowList = const <String, int>{},
+    this.allowAllNotDisposed = false,
+    this.allowAllNotGCed = false,
+  });
+
+  /// Creates a new instance for debugging notDisposed leaks.
+  ///
+  /// This configuration will collect stack traces on start and disposal,
+  /// and retaining path for notGCed objects.
```

Comment:
"retaining" => "retain"

Reply:
`retaining path` is a concept in memory management: https://docs.flutter.dev/tools/devtools/memory#root-object-retaining-path-and-reachability

replaced with 'the retaining path' to show it is noun.

Reply:
Ah got it, thanks!